src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+from django.db import IntegrityError, transaction
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.bases.dashboard import DashboardSerializer
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.models import Dashboard
+from rest_framework.response import Response
+
+
+class OrganizationDashboardsEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization):
+        """
+        Retrieve an Organizations Dashboards
+        `````````````````````````````````````
+        Retrieve a list of dashboards that are associated with the given organization.
+        :pparam string organization_slug: the slug of the organization the
+                                          dashboards belongs to.
+        :qparam string query: the title of the dashboard being searched for.
+        :auth: required
+        """
+        dashboards = Dashboard.objects.filter(
+            organization_id=organization.id
+        )
+        query = request.GET.get('query')
+        if query:
+            dashboards = dashboards.filter(
+                title__icontains=query,
+            )
+
+        return self.paginate(
+            request=request,
+            queryset=dashboards,
+            order_by='title',
+            paginator_cls=OffsetPaginator,
+            on_results=lambda x: serialize(x, request.user),
+        )
+
+    def post(self, request, organization):
+        """
+        Create a New Dashboard for an Organization
+        ``````````````````````````````````````````
+        Create a new dashboard for the given Organization
+        :pparam string organization_slug: the slug of the organization the
+                                          dashboards belongs to.
+        :param string title: the title of the dashboard.
+        :param user created_by: the user who created the dashboard
+                                this can be represented by the user's (verified) email
+                                or the user's id.
+        """
+        serializer = DashboardSerializer(data=request.DATA)
+
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+
+        result = serializer.object
+
+        try:
+            with transaction.atomic():
+                dashboard = Dashboard.objects.create(
+                    organization_id=organization.id,
+                    title=result['title'],
+                    created_by=result['created_by'],
```

Comment:
Should a user be able to create dashboards as someone else? It would be safer to use the session/currently active user to create author information.